R/Flag_Poisson.R

```diff
@@ -6,7 +6,7 @@
 #' Add columns flagging sites that represent possible statistical outliers when the Poisson statistical method is used.
 #'
 #' @details
-#' This function flags sites based on the Poisson analysis result as part of the [GSM data pipeline](https://gilead-biostats.github.io/gsm/articles/DataPipeline.html).
+#' This function flags sites based on the Poisson analysis result as part of the [GSM data model](https://gilead-biostats.github.io/gsm/articles/DataModel.html).
```

Comment:
```suggestion
#' This function flags sites based on the Poisson analysis result as part of `vignette("DataModel")`.
```
Will auto-link + work offline, per R Packages (more details in vignette reviews).